docs/sources/enterprise/license/activate-aws-marketplace-license/activate-license-on-ecs.md

```diff
@@ -10,70 +10,95 @@ weight = 250
 
-To activate your license, complete the following tasks:
+To activate your license, complete the following tasks.
 
 ## Task 1: Deploy Grafana Enterprise on Amazon ECS
 
-If you do not have Grafana Enterprise running in ECS already, you must deploy it. Follow Amazon's documentation to [Create an Amazon ECS service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create-service.html). While creating the service, use the container image for Grafana Enterprise version 8.3.0 or higher. For example, use `grafana/grafana-enterprise:8.3.3`.
+1. Deploy Grafana Enterprise on Amazon ECS.
 
-Versions of Grafana before 8.3.0 do not support licenses granted through AWS Marketplace.
+   For more information about deploying an application on Amazon ECS, refer to [Creating an Amazon ECS service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create-service.html).
 
-## Task 2: Setup Grafana for high-availability
+1. As you create the Amazon ECS service, use the Grafana Enterprise version 8.3.0 or later container image.
 
-Update the [database]({{< relref "../../../../../administration/configuration.md#database" >}}) configuration section, so that Grafana use a shared database for storing dashboard, users, and other persistent data.
+   For example, enter `grafana/grafana-enterprise:8.3.3`.
 
-> If you do not have a MySQL database already, [create an Amazon RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) and check the information needed to [connect to your RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_CommonTasks.Connect.html).
+   > Only Grafana Enterprise versions 8.3.0 and later support licenses granted through AWS Marketplace.
 
-Create a new revision of the Task Definition for the ECS Task that runs Grafana Enterprise. For details, refer to the AWS documentation on [updating a task definition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/update-task-definition.html).
+## Task 2: Configure Grafana for high availability
 
-Within the new revision, edit the Grafana Enterprise container for this task, and add the following environment variable to the container:
+Grafana requires that you configure a database to hold dashboards, users, and other persistent data.
 
-```
-GF_DATABASE_TYPE=mysql
-GF_DATABASE_HOST=localhost:3306
-GF_DATABASE_NAME=grafana
-GF_DATABASE_USER=[mysql database username]
-GF_DATABASE_PASSWORD=[mysql database password]
-```
+### Before you begin
 
-> For more information about how to update your ECS service with an environment variable, refer to [Updating a service using the new console](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/update-service-console-v2.html).
+- Ensure that you have a MySQL database available. For information about creating a MySQL database, refer to [Creating an Amazon RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html).
```

Comment:
The database doesn't necesseraly need to be MySQL, it can be any of the supported types by Grafana, at this moment MySQL or Postgres.

Reply:
Took care of this one. Added a link to our list of supported dbs and removed references to MySQL.